hack/dev-release.sh

```diff
@@ -3,14 +3,28 @@
 set -e
 set -u
 
-OPERATOR_SOURCE="./test/operator-hub/operator_source.yaml"
+INDEX_IMAGE="${INDEX_IMAGE:-quay.io/$OPERATOR_GROUP/$GO_PACKAGE_REPO_NAME-index:$BUNDLE_VERSION}"
```

Comment:
Do we ever plan to run this script standalone. i.e. without Makefile?

Reply:
probably not...